<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset theme="zoom-fit-best">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>970</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Determinante:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="a31">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a23">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a13">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a33">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QPushButton" name="boton1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="a12">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Determinante de matriz 3x3</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="determinante">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="a32">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a21">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a11">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a22">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Determinante:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="a23_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a33_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QPushButton" name="boton2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="a31_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a41_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a32_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a43_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a22_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a12_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a21_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a44_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a14_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a34_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>170</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Determinante de matriz 4x4</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="a11_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a42_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a13_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="determinante_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>310</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="a24_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Determinante:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="a22_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a13_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>410</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a31_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>510</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a33_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>510</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a23_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>460</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QPushButton" name="boton4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>570</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="a12_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>350</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ecuaciones 3x3</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="a32_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>510</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>600</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Soluciones:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="a11_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>410</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="solucion2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>600</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="a21_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>460</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="igual_1">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>410</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="igual_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>460</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="igual_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>510</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>410</y>
      <width>21</width>
      <height>141</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="solucion1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>570</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Soluciones:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>540</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="a22_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>480</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="igual_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="igual_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>21</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="a21_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QTextEdit" name="a12_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>420</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ecuaciones 2x2</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="a11_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>16</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>16</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>16</width>
      <height>201</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>16</width>
      <height>201</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_borrar1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_borrar2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>280</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_borrar3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>540</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_borrar4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>570</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>-60</x>
      <y>-10</y>
      <width>811</width>
      <height>661</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Pictures/hand-drawn-physic-formulas-science-knowledge-education-chem-formula-and-physics-math-formula-and-physics-white-background-hand-drawn-line-math-formula-and-physics-formula-vector.jpg</pixmap>
    </property>
   </widget>
   <zorder>label_10</zorder>
   <zorder>label_4</zorder>
   <zorder>a31</zorder>
   <zorder>a23</zorder>
   <zorder>a13</zorder>
   <zorder>a33</zorder>
   <zorder>boton1</zorder>
   <zorder>a12</zorder>
   <zorder>label_2</zorder>
   <zorder>determinante</zorder>
   <zorder>a32</zorder>
   <zorder>a21</zorder>
   <zorder>a11</zorder>
   <zorder>a22</zorder>
   <zorder>label</zorder>
   <zorder>a23_2</zorder>
   <zorder>a33_2</zorder>
   <zorder>boton2</zorder>
   <zorder>a31_2</zorder>
   <zorder>a41_2</zorder>
   <zorder>a32_2</zorder>
   <zorder>a43_2</zorder>
   <zorder>a22_2</zorder>
   <zorder>a12_2</zorder>
   <zorder>a21_2</zorder>
   <zorder>a44_2</zorder>
   <zorder>a14_2</zorder>
   <zorder>a34_2</zorder>
   <zorder>label_3</zorder>
   <zorder>a11_2</zorder>
   <zorder>a42_2</zorder>
   <zorder>a13_2</zorder>
   <zorder>determinante_2</zorder>
   <zorder>a24_2</zorder>
   <zorder>label_5</zorder>
   <zorder>a22_3</zorder>
   <zorder>a13_3</zorder>
   <zorder>a31_3</zorder>
   <zorder>a33_3</zorder>
   <zorder>a23_3</zorder>
   <zorder>boton4</zorder>
   <zorder>a12_3</zorder>
   <zorder>label_6</zorder>
   <zorder>a32_3</zorder>
   <zorder>label_7</zorder>
   <zorder>a11_3</zorder>
   <zorder>solucion2</zorder>
   <zorder>a21_3</zorder>
   <zorder>igual_1</zorder>
   <zorder>igual_2</zorder>
   <zorder>igual_3</zorder>
   <zorder>line</zorder>
   <zorder>solucion1</zorder>
   <zorder>label_8</zorder>
   <zorder>boton3</zorder>
   <zorder>a22_4</zorder>
   <zorder>igual_4</zorder>
   <zorder>igual_5</zorder>
   <zorder>line_2</zorder>
   <zorder>a21_4</zorder>
   <zorder>a12_4</zorder>
   <zorder>label_9</zorder>
   <zorder>a11_4</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>line_5</zorder>
   <zorder>line_6</zorder>
   <zorder>boton_borrar1</zorder>
   <zorder>boton_borrar2</zorder>
   <zorder>boton_borrar3</zorder>
   <zorder>boton_borrar4</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
